doc/sphinxman/source/scf.rst

```diff
@@ -579,17 +579,23 @@ that |PSIfour| expects the numpy file on disk to have the ``.npy`` extension, no
 Convergence Stabilization
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-With regard to convergence stabilization, Pulay's Direct Inversion of the
-Iterative Subspace (DIIS) extrapolation,  Gill's Maximum Overlap Method (MOM),
-and damping are all implemented. A summary of each is presented below,
+A summary of Psi's supported convergence stabilization techniques is presented below:
 
 DIIS [On by Default]
-    DIIS uses previous iterates of the Fock Matrix together
+    DIIS uses previous iterates of the Fock matrix together
     with an error criterion based on the orbital gradient to produce an informed
     estimate of the next Fock Matrix. DIIS is almost always necessary to converge
     the SCF procedure and is therefore turned on by default. In rare cases, the
     DIIS algorithm may need to be modified or turned off altogether, which may be
     accomplished via :term:`options <DIIS (SCF)>`.
+ADIIS [On by Default]
+    ADIIS uses previous iterates of the Fock and density matrices to produce an
+    informed estimate of the next Fock matrix. ADIIS estimates are based on minimizing
+    the energy rather than zeroing the residual, so this performs best in the early
+    iterations. By default, Psi will start using ADIIS before blending the ADIIS step with
+    the DIIS step, eventually using the pure DIIS step. The closely-related EDIIS procedure
+    may be used instead by setting |scf__initial_scf_acelerator|. This is formally identical
+    to ADIIS for HF, but will differ for more general DFT.
```

Comment:
but the methods differ for DFT

Reply:
This is again obsessively technical, but some people are insistent than HF counts as a DFT functional. Reworded to "but the methods will differ for more general DFT."

Reply:
But the argument that HF counts as DFT is not clear in this sentence => "can differ for DFT".

Reply:
This disagreement is more technical than is appropriate for this section of the documentation to get into, and whichever option we choose is ultimately unimportant. Rejected.